index.html

```diff
@@ -1,57 +1,194 @@
-            </li>
-            <li>
-                <img src="imgs/Linkedin-icon.svg" alt="Linkedin img">
-            </li>
-            <li>
-                <img src="imgs/medium.svg" alt="Medium img">
-            </li>
-            <li>
-                <img src="imgs/github.svg" alt="Github img">
-            </li>
-            <li>
-                <img src="imgs/angellist.svg" alt="Angellist img">
-            </li>
-            </ul>
-        </div>
-    </div>
-    </section>
-    
+	<header>
+	<nav class="nav-bar">
+		<a href="#" onClick="return false;">
+		<p class="title">Welcome</p>
+		</a>
+		<a class="icon">
+			<i class="fa fa-bars"></i>
+		</a>
+	</nav>
+	</header>
+	<section class="main-container">
+		<div class="headline">
+			<div class="intro-txt">
+				<h1 class="wave-txt">Hey there. I'm Lucas</h1>
+				<h2 class="desc-txt">I'm a software developer</h2>
+			</div>
+			<div class="main-headline-txt">
+				<p>
+					I can help you build a product, feature or website Look through some of my work and experience! 
+					If you like what you see and have a project you need coded, don't hestiate to contact me.
+				</p>
+			</div>
+			<div class="media-imgs">
+				<ul>
+				<li>
+					<img src="imgs/twitter.svg" alt="Twitter img">
+				</li>
+				<li>
+					<img src="imgs/Linkedin-icon.svg" alt="Linkedin img">
+				</li>
+				<li>
+					<img src="imgs/medium.svg" alt="Medium img">
+				</li>
+				<li>
+					<img src="imgs/github.svg" alt="Github img">
+				</li>
+				<li>
+					<img src="imgs/angellist.svg" alt="Angellist img">
+				</li>
+				</ul>
+			</div>
+		</div>
+	</section>
+	<section class="works-sec">
+		<div class="works-title-flex">
+			<h2 class="wave-txt" id="works-txt-id">My Recent Works</h2>
+			<img class="div-img" src="imgs/indicator.png" alt="div-img">
+		</div>
+		<div class="main-container-works">
+			<div class="grid-container-works">
+				<div class="card-flex-container">
+					<div class="grey-bckgd"></div>
+					<div class="card-contents">
+					<div class="card-subt">
+						<h3>Multi-Post Stories Gain+Glory</h3>
+					</div>
+					<div class="card-buttons">
+						<button class="lang-button">Ruby on rails</button>
+						<button class="lang-button">css</button>
+						<button class="lang-button">JavaScript</button>
+						<button class="lang-button">html</button>
+					</div>
+					<div class="card-see-prjct">
+						<button class="green-button">See Project</button>
+					</div>
+					</div>
+				</div>
+			</div>
+			<div class="grid-container-works">
+				<div class="card-flex-container">
+					<div class="grey-bckgd"></div>
+					<div class="card-contents">
+					<div class="card-subt">
+						<h3>Multi-Post Stories Gain+Glory</h3>
+					</div>
+					<div class="card-buttons">
+						<button class="lang-button">Ruby on rails</button>
+						<button class="lang-button">css</button>
+						<button class="lang-button">JavaScript</button>
+						<button class="lang-button">html</button>
+					</div>

```

Comment:
- [ ] Please use `ul` and `li` tags for the list of technologies. Because these technologies are not going to be clicked like a button. They are just lists of texts with background styling.